### DISCUSSION TITLE
Add Book able products.

### DISCUSSION THREAD

**Discussion by pratikrane1** [inspireui/support#5582] [New Feature/Ideas Requests] [answered]
How to Show booked able products in Product page. I am using  Woocommerce Booking plugin(https://woocommerce.com/products/woocommerce-bookings/). But, I am unable to add booked able product in application. Can you help me how to add cooked able products?

**Comment by bakpaorabbit** [comment, marked as answer]
you need extended license to get the fluxtsore app with booking source code. It support BookingWp plugin.
